common/hashmap.cpp

```diff
@@ -58,11 +58,11 @@ template<> void unknownKeyError(::Common::String key) {
 }
 
 template<> void unknownKeyError(signed char key) {
-	error("Unknown key \"%hhi\"", key);
+	error("Unknown key \"%d\"", int(key));
```

Comment:
I would say that specifying int and unsigned is useless as key will be promoted to int.
cf. https://en.cppreference.com/w/cpp/language/variadic_arguments
>  bool, char, short, and unscoped enumerations are converted to int or wider integer types as in integer promotion

Reply:
Done